### ISSUE TITLE
Bulk renaming doesn't work. Doing it wrong?

### ISSUE THREAD

**Issue by dadaphl** [jarun/nnn#1340]
#### Environment details (Put `x` in the checkbox along with the information)

- [X] Operating System: macOs 12.1 (M1) installed with port
- [ ] Desktop Environment:
- [X] Terminal Emulator: iterm2
- [X] Shell: zsh
- [ ] Custom desktop opener (if applicable):
- [ ] Program options used:
- [ ] Configuration options set:
- [ ] Plugins are installed
- [ ] Issue exists on `nnn` master

#### Exact steps to reproduce the issue

- open nnn in directory
- press r, vim opens
- edit file names, 
- save (:wq)
files still have old name

I couldn't find a explanation how bulk editing should work in the docs so I assume that the steps I described here should do it. Nevertheless  the files still have the old name. Am I doing it wrong?

**Comment by N-R-K** [comment]
> edit file names,

Do you see numbers on the left side of the file? If so, you aren't supposed to edit the numbers.

**Comment by dadaphl** [comment]
no, i just edit the file name.

**Comment by 0xACE** [comment]
Are you removing any lines? The bulk renamer exacts each file to have its own corresponding line. If a line is removed the bulk renamer refuses to rename because it cannot know which line you removed and what corresponds to what…

**Comment by dadaphl** [comment]
No, i'm really just editing the name of the file (just remove one char).
I have $EDITOR set to nvim (neovim). I noticed that it doesn't seem to work properly. It still opens vim with 
`Vim: Warning: Input is not from a terminal`
I use zsh in tmux. Maybe I have something mis-configured.
I also noticed that the vim buffer result (the renamed list of files) got written (overwrite) to one of the files in the directory. Very strange.
